lib/createJestRunner.js

```diff
@@ -73,7 +73,17 @@ const createRunner = (runPath, { getExtraOptions } = {}) => {
                   return runner(baseOptions);
                 });
               })
-              .then(result => onResult(test, result))
+              .then(testResult => {
+                if (Array.isArray(testResult)) {
+                  testResult.forEach(result =>
+                    result.numFailingTests > 0
```

Comment:
Can you do an `if` rather than the ternary?

Reply:
ok, soon maybe